standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```diff
@@ -134,7 +135,7 @@ public static String convertTimestampToString(Timestamp timestamp) {
    */
   public static Timestamp convertStringToTimestamp(String timestamp) {
     LocalDateTime val = LocalDateTime.from(TIMESTAMP_FORMATTER.parse(timestamp));
```

Comment:
Since now we are converting to `Instant` do we need to pass from `LocalDateTime`? Can we use directly `Instant.from` here?